eZ/Publish/Core/IO/IOMetadataHandler/LegacyDFSCluster.php

```diff
@@ -60,6 +56,11 @@ public function create(SPIBinaryFileCreateStruct $binaryFileCreateStruct)
     {
         $path = $this->addPrefix($binaryFileCreateStruct->id);
 
+        // Some code has been using integer, though DateTime is correct. Convert to DateTime if needed.
```

Comment:
@andrerom Ok, sure. In that case maybe we should throw an `InvalidArgumentException` if the mtime isn't a `DateTime`? Bit of a BC break. Anyway let's see if all tests pass first.